node_modules/col-users/lib/rest.js

```diff
@@ -19,5 +19,12 @@ var Collabosphere = require('col-core');
  * The me feed
  */
 Collabosphere.apiRouter.get('/me', function(req, res) {
-  return res.status(200).send(req.ctx.user);
+  if (!req.ctx) {
+    return res.status(401).send('Unauthenticated API request');
+  }
+
+  // Add the admin status to the response
+  var me = req.ctx.user.toJSON();
+  me.is_admin = req.ctx.user.isAdmin();
```

Comment:
I wonder if it makes sense to define a special getter for isAdmin (I believe they get included when serializing an object) or maybe override `User.toJSON`